src/cleanup_rules/swift/rules.toml

```diff
@@ -131,6 +133,49 @@ query = """(
 groups = ["boolean_expression_simplify"]
 replace_node = "disjunction_expression"
 replace = "@lhs"
+#
+# Before 
+#   false && abcd()
+# After 
+#   false
+#
+[[rules]]
+name = "false_and_something"
```

Comment:
```suggestion
name = "false_and_anything"
```

Reply:
I think since we are using `(_)`  (the wild card pattern) we should call it `anything` rather than `something`. I know this is legacy naming convention. But just something i thought about when reviewing.

Reply:
We can have a separate PR which renames `something` to `anything` when necessary too.

Reply:
I will create a seprate PR to resolve this since alot of code with `something` has landed